app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+   has_and_belongs_to_many :users  
```

Comment:
same post cant  have many User,
it should be user has_many posts

Reply:
Thanks, Eng but I use it to see how I can use Many to Many relations in Rails